packages/integrations/sitemap/src/write-sitemap.ts

```diff
@@ -64,6 +66,6 @@ export async function writeSitemap(
 	});
 
 	let src = Readable.from(sourceData);
-	const indexPath = resolve(destinationDir, `./sitemap-index.xml`);
```

Comment:
not sure if we want to change this at all but if we set `separator: ""` then it'd be `sitemapindex.xml`.

This makes sense for `sitemap-0.xml` => `sitemap0.xml` but not sure about the index?

happy to change anything if we need to.